### PR TITLE
[Bugfix] Fix displaying snapshots of Paimon tables

### LINKED ISSUE #245: [Bug] Can not display paimon table snapshot
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/paimon-webui/issues) and found nothing similar.

### Paimon UI version

main

### Compute Engine

Flink

### Minimal reproduce step

image

### What doesn't meet your expectations?

The snapshot is displayed correctly

### Anything else?

_No response_

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by hdygxsj**
### Purpose

Fixed the issue with displaying snapshots of Paimon tables. close #245 
image

### CODE REVIEW

**Comment by SteNicholas** [comment]
@hdygxsj, please create new issue to record this bug.

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java [hunk 1]
```diff
@@ -120,9 +120,7 @@ public List<SnapshotVO> getSnapshot(MetadataDTO dto) {
                                         .totalRecordCount(internalRow.getLong(9))
                                         .deltaRecordCount(internalRow.getLong(10))
                                         .changelogRecordCount(internalRow.getLong(11))
-                                        .addedFileCount(internalRow.getInt(12))
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java:123]: Why remove setting added file count and deleted file count?

**Comment by hdygxsj** [comment]
Because paimon0.8 removed them



---Original---
From: "Nicholas ***@***.***&gt;
Date: Fri, May 24, 2024 16:35 PM
To: ***@***.***&gt;;
Cc: "yangyang ***@***.******@***.***&gt;;
Subject: Re: [apache/paimon-webui] [Bugfix] Fix displaying snapshots of Paimontables (PR #242)




 
@SteNicholas commented on this pull request.
 
 
In paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java:
 &gt; @@ -120,9 +120,7 @@ public List<SnapshotVO&gt; getSnapshot(MetadataDTO dto) {                                          .totalRecordCount(internalRow.getLong(9))                                          .deltaRecordCount(internalRow.getLong(10))                                          .changelogRecordCount(internalRow.getLong(11)) -                                        .addedFileCount(internalRow.getInt(12))  
Why remove setting added file count and deleted file count?

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/data/vo/SnapshotVO.java [hunk 2]
```diff
@@ -56,9 +56,9 @@ public class SnapshotVO {
 
     private Long changelogRecordCount;
 
-    private Integer addedFileCount;
+    private Long addedFileCount;
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/data/vo/SnapshotVO.java:59]: Why not remove `addedFileCount` and `deletedFileCount`?

**Review by SteNicholas — APPROVED** [review]
LGTM.

**Pull request merged**